app/views/application/faq.html.erb

```diff
@@ -26,7 +26,9 @@
       Taxpayer Identification Number (TIN). If you do not have an active DUNS
       number, you may obtain one online at the <a
         href="http://fedgov.dnb.com/webform">Dun &amp; Bradstreet site</a>. This
-      process takes one-to-two business days. If you do not have a TIN, you may
+      process takes one-to-two business days, and the Dun &amp; Bradstreet website
+      is not compatible with Google Chrome: Safari or Internet Explorer are recommended.
```

Comment:
I'm using a Mac. Chrome `53.0.2785.80 beta (64-bit)`, uBlock + HTTPS everywhere on. Called DUNS and they immediately said to avoid using Chrome.